src/modules/Articles.js

```diff
@@ -1,40 +1,53 @@
 import axios from 'axios';
 import Popup from './Popup';
+import store from '../state/store/configureStore';
+import errorHandler from './ErrorHandler'
 
 const Articles = {
   async create(event, category, setModalOpen) {
-    let headers = getFromLocalStorage()
     let params = {
       article: {
         title: event.target.title.value,
         teaser: event.target.teaser.value,
         body: event.target.body.value,
         category: category,
       },
-    }
+    };
     try {
-      let response = await axios.post('/articles', { params: params }, { headers: headers });
-      Popup.open('SUCCESS_MESSAGE', response.data.message);
+      let response = await axios.post('/articles', params, {
+        headers: getFromLocalStorage(),
+      });
       setModalOpen(false);
+      Popup.open('SUCCESS_MESSAGE', response.data.message);
     } catch (error) {
-      errorHandler(error)
+      errorHandler(error);
+    }
+  },
+
+  async index() {
+    try {
+      let response = await axios.get('/articles', {
+        headers: getFromLocalStorage(),
+      });
+      if (response.status === 204) {
+        store.dispatch({
+          type: 'SET_ARTICLES',
+          payload: "You don't have any articles yet",
+        });
```

Comment:
Could be seen as contamination to change the type of articles to a string. 
Alternatively, we could just set it as an empty array, and use that as the conditional instead. 
That also prevents us of having to use a function inside the render

Reply:
Good point - changed that!